### DISCUSSION TITLE
How to configure desktop notifications

### DISCUSSION THREAD

**Discussion by plantarum** [i3/i3#6303] [Q&A]
Hi,

I'm a happy i3 user, on Ubuntu 20.04 and also Debian. My system notifications show up in the top right corner, in a box like this:

![notificaiton_000](https://github.com/user-attachments/assets/dca4be33-6553-4001-8d72-b08e6da4ee07)

Does anyone know what utility generates these little windows, and how to customize them? I have `libnotify4` installed, a long with `gir1.2-notify-0.7`, but the `notify-send` binary isn't present that I can find.

**Comment by okraits** [comment]
See https://dunst-project.org/
